packages/react/src/components/TableHandles/mantine/TableHandle.tsx

```diff
@@ -2,20 +2,75 @@ import {
   DefaultInlineContentSchema,
   DefaultStyleSchema,
   InlineContentSchema,
+  mergeCSSClasses,
   StyleSchema,
 } from "@blocknote/core";
-import { MdDragIndicator } from "react-icons/md";
+import { Menu } from "@mantine/core";
+import { ReactNode, useState } from "react";
 
 import { TableHandleProps } from "../TableHandleProps";
-import { TableHandleWrapper } from "./TableHandleWrapper";
+import { MdDragIndicator } from "react-icons/md";
+import { TableHandleMenu } from "../TableHandleMenu/mantine/TableHandleMenu";
 
+// TODO: props.tableHandleMenu should only be available if no children are
```

Comment:
why?

Reply:
Same as side menu - if consumer implements their own drag handle (custom or using default component) they will already define the menu there so it's useless to pass it in as an additional prop.